controllers/operands/kubevirt_test.go

```diff
@@ -1032,6 +1032,119 @@ Version: 1.2.3`)
 			})
 		})
 
+		Context("test CPUModel", func() {
+
+			It("should propagate the CPUModel from the HC if set", func() {
+				existKv, err := NewKubeVirt(hco)
+				Expect(err).ToNot(HaveOccurred())
+
+				testCPUModel := "testValue"
+				hco.Spec.DefaultCPUModel = &testCPUModel
```

Comment:
Please consider to use the `pointer` package; e.g. 
```suggestion
				hco.Spec.DefaultCPUModel = pointer.String("testValue")
```

Reply:
done